src/Authentication/Authentication.Core/Microsoft.Graph.Authentication.Core.csproj

```diff
@@ -11,7 +11,7 @@
     <EnforceCodeStyleInBuild>true</EnforceCodeStyleInBuild>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="Azure.Identity" Version="1.10.2" />
+    <PackageReference Include="Azure.Identity" Version="1.9.0" />
```

Comment:
The failing check points to a vulnerability with `1.9.0`. You want to use `1.10.3` instead.

To fix the error noted in v1.10.2, you may need to use `Azure.Identity.Broker v1.0.0-beta.5`. See my suggestion below.